dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -5,7 +5,7 @@
 - location = Geocoder.search(request.ip).try(:first)
 - country_code = location&.country_code.to_s.upcase
 - state_code = location&.state_code.to_s.upcase
-- us_ip = ['US', 'RD', nil].include?(country_code)
```

Comment:
Note: I added `, nil` option in one of my early PR's. I was thinking that we should default to our US policy, but now I don't think we should always apply US based policies if we don't know where they live. Assuming where they live would be inconsistent with how we treat EU students. Fore example, if we don't detect a student's country, we don't assume the GDPR applies to them.

Reply:
Removing it feels like the right idea. Then, we aren't surprised when this gets randomly applied.

And I don't think I would have actually understood that this was the purpose of the `nil` here. Actually, I think I'm ignorant of what the `RD` refers to. Perhaps that's worth documenting.

Reply:
I'm not 100% sure, but the in[ the geocoder unit tests](https://github.com/alexreisner/geocoder/blob/2dc9254620e076fd3a70f95f78d8760f5114f4e5/test/unit/lookups/ipbase_test.rb#L33), "Reserved" is full country name they have associated with it. It's the country code returned when the given IP address is `localhost`.

Reply:
I think I have written that down in another part of our codebase, but I will write it here too.

Reply:
Ah! That makes sense. And makes sense why google wasn't helping me find the "RD" country. I appreciate the documenting... because this was definitely a "TIL" moment.